content/md/products-super-power/improve-data-quality.md

```diff
@@ -4,65 +4,89 @@ themes: products-super-power
 
-# Measure data quality
+### Key indicators
+Key indicators deep-dive into one particular criteria that needs to be fulfilled to reach a perfect quality score.
 
-From the dashboard, you can see at a glance the current grades for your favorite categories and families.
+- **Products with an image**: enables you to know the percentage of products with at least 1 image and also how many products need an image to fulfill the criteria.
+- **Products with a good enrichment ratio**: enables to know the percentage of products with at least 80% of enriched attributes and how many products need to be enriched further.
+- **Products with perfect spelling in text product values**: enables you to know the percentage of products without spelling mistakes in the product data and how many products hold correction suggestions for spell-check.
+- **Products properly labelled**: enables you to know the percentage of attributes and options without spelling mistakes and how many hold correction suggestions for spell-check.
+
+::: success
+By clicking on the number of products that need to be worked on, you are led to a product grid with all products that need to be improved on this specific criteria.
+:::
+
+### Categories and families
+
+From the dashboard, you can see at a glance the current quality score for your favorite categories and families.
 
 ![Image data quality widgets](../img/data-quality-widgets.png)
 
 To do so, it’s very easy:
 1. Click on `Select a category` or `Select a family` from the widgets
 2. Choose the categories and families you want to add to your dashboard
-3. See their enrichment and consistency grades
+3. See their quality score
 
 ::: success
 The cherry on the cake: click on `See on the grid` to access a filtered view of your catalog to focus your work on what needs most improvements!
 :::
 
 
 # Improve data quality
-Now that you know how to measure your catalog’s data quality, it’s easier to know which families and products need to be worked on.
-To help you improve your products **enrichment** and **consistency**, simply follow the **recommendations** that can be found in the **data quality insights** panel in your product edit form.
+Now that you know how to measure and follow improvements on your catalog’s data quality, it’s easier to know which families and products need to be worked on.
+To help you improve your products **quality score**, simply follow the **recommendations** that can be found in the **data quality insights** panel in your product edit form.
 
 ![Image data quality panel](../img/data-quality-panel.png)
 From the `data quality panel`, you will know which attributes need to be improved and how to improve them.
 
-::: info
+::: tips
 From the `data quality panel`, links on attribute names can take you to a specific attribute. By clicking on the `Display all improvable attributes`, you can easily access a product edit form filtered on all the attributes that need to be checked for consistency.
 :::
```

Comment:
::: tips
From the `data quality panel`, links on attribute names can take you to a specific attribute. By clicking on `Display all improvable attributes`, you can easily access a product edit form filtered on all the attributes that need to be checked for consistency.
:::